### PR TITLE
ncnews6

### PR DESCRIPTION
**Pull request by Castelli87**

### CODE REVIEW

**Review by HannahBurks — COMMENTED** [review]
We need to put txt files and playground.sql in a git ignore and delete from github

**Review by HannahBurks — CHANGES_REQUESTED** [review]
For path /api/articles/:article_id/comments

TESTS:
- Would like you to check the article ID is as expected ( /api/articles/1/comments - article_id would be 1) 
- Would also like you to check the order of the comments in some way - most recent comment first (jest sorted?)
- Add test for Status 200, valid ID, but has no comments responds with an empty array of comments

File: __tests__/app.test.js [hunk 1]
```diff
@@ -89,6 +89,42 @@ describe("GET - /api/articles/invalidArticleId", () => {
       .expect(404)
       .then((response) => {
         expect(response.body.msg).toBe("not found");
+
+      });
+  });
+});
+
+describe.only("/api/articles", () => {
```
[the remark this answers is not in the document]
Comment (Castelli87) [reply __tests__/app.test.js:97]: - So i fixed the first point so checking the article_id tobe 1;
- i  fixed the second bullet point 
- i don't know what i need to do for this bullet point because i have the logic in the model.js that if the length is 0 ew reject the promises and send back a custom error 

the .only appear only in the main branch and not in the ncnews6
